docs/maya/tools.md

```diff
@@ -120,6 +120,8 @@ In the specific case of an AdnSkin deformer, the tool will display the following
     - This value is normalized alongside Soft Constraints and Slide Constraints.
   - **Masses**
     - Masses are set to 1.0 by default. This will mean that by default the solver will consider that the skin has a uniform mass.
+  - **Shape Preservation**
+    - Shape preservation weights are set to 0.0 by default. Modify this value to allow the solver to apply corrections to the current vertex to maintain the initial state of the 3 vertices conforming a single shape preservation constraint. 
```

Comment:
Is default to 0.0 in some deformers and to 0.1 in others.

I would not mention only three vertices but the surrounding. Also I would not mention single shape preservation constraint. If I'm not wrong more than one is created per vertex.